src/main/java/com/example/OptionalTask.java

```diff
@@ -0,0 +1,25 @@
+//2.Вывести n чисел с консоли. Вывести числа в порядке возрастания значений их длины.
```

Comment:
Please avoid pushing off-topic files into your pull-request:
Your work in this pull request is about Main tasks, not Optional ones.
So please remove this file from this pull-request and create new pull-request for optional tasks.